woocommerce/woocommerce-android
# [Deposit Summary] Tracking

Closes: #10071


### Description


* `PAYMENTS_HUB_DEPOSIT_SUMMARY_SHOWN` with property `number_of_currencies: {number of currencies returned}` Tracked every time the deposit summary is displayed. It may be beneficial to know the number of currencies given to better understand the profile of the merchants that may benefit from this feature the most.
* `PAYMENTS_HUB_DEPOSIT_SUMMARY_EXPANDED`. It is tracked every time a user clicks on the show summary, and more information is displayed. So we can understand if people understand the “collapse/expand” mechanism, and we can add the future “instant deposit” button inside.
* `PAYMENTS_HUB_DEPOSIT_SUMMARY_ERROR` with the property `error_description: {}` . Tracked every error returned from the API
* `PAYMENTS_HUB_DEPOSIT_SUMMARY_LEARN_MORE_CLICKED`. It is tracked every time when learn more is clicked. To see if this is needed or if we can remove that in the future

### Testing instructions

Use logcat and notice the new events

e.g.:

```
Tracked: payments_hub_deposit_summary_learn_more_clicked, Properties: {"blog_id":217705087,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/thebritwootester.wpcomstaging.com"}
Tracked: payments_hub_deposit_summary_expanded, Properties: {"blog_id":217705087,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/thebritwootester.wpcomstaging.com"}
Tracked: payments_hub_deposit_summary_shown, Properties: {"number_of_currencies":"1","blog_id":217705087,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/thebritwootester.wpcomstaging.com"}
Tracked: payments_hub_deposit_summary_error, Properties: {"error_context":"PaymentsHubDepositSummaryViewModel","error_description":"Error: No account found for this site.","error_type":"API_ERROR","blog_id":192152755,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/myusernametestsite2020151673500.in"}
```

### Images/gif
